packages/react-components/src/input/src/Input.css

```diff
@@ -188,7 +187,7 @@
 }
 
 /* BUTTON */
-.o-ui-input-button {
+.o-ui-input-has-button .o-ui-input-button {
```

Comment:
Not sure why would need to add `has-button` here? The class `o-ui-input-button` would only be available there IS a button. There IMO it's redundant and useless.